sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Utils.java

```diff
@@ -548,4 +549,12 @@ public ValueHolder(V v) {
         holder.v = dictionary.remove(key);
         return holder.v != null;
     }
+
+    public static <T> T parse(String itemResponseBodyAsString, Class<T> itemClassType) {
```

Comment:
I looked up and there are 4 places where we use `Utils.getsimpleObjectMapper().readValue()` APIs. 
It would be great if we can consolidate them here and have this logic at one place.

Reply:
Will check and do the refactoring as required in a different PR as this is not directly related to this.